beta/project-griffon/using-project-griffon/adobe-analytics-and-project-griffon.md

```diff
@@ -6,7 +6,21 @@ The integration with Adobe Analytics provides a richer view of SDK events to use
 
 ![](../../../.gitbook/assets/aa-loop.gif)
 
-If the SDK made a network request with Adobe Analytics \(queued\), and if post-processing information is returned from Adobe Analytics about that event \(validated\), the status tells you when an SDK event is generated \(processed\).
+### Post-processed status
+
+If the SDK made a network request with Adobe Analytics, and if post-processing information is returned from Adobe Analytics about that event, the status tells you when an SDK event is generated.
+
+| Status | Description |
+| :--- | :--- | 
+|`Queued`|Network request is being made to fetch the post-processing information.|
+|`Processed`|Network request is successful and post-processing information is received.|
+|`Delayed`|Exceeded the maximum network request retries to fetch the post-processing information.|
+|`Error`|Error casued the network request to fail. More details about the error is displayed in the event details view.|
+|`Unauthorized`|Failed to retrieve post-processed data because the user does not have access to the Analytics report suite.|
+|`Unavailable`|`AnalyticsTrack` or `LifecycleStart` event does not have corresponding `AnalyticsResponse` event.|
```

Comment:
"Neither the `AnalyticsTrack` nor the `LifecycleStart` event have a corresponding `AnalyticsResponse` event."

Reply:
These are two separate kinds of events that we show in the list, each have their own response event. Should we say `The AnalyticsTrack or the LifecycleStart event does not have a corresponding AnalyticsResponse event.` or keep it generic and instead say. `The Analytics request does not have a corresponding AnalyticsResponse event.`?

Reply:
Ahhh, I see, I see. I'd go with something more generic then, and be like "The Adobe Analytics request does not have a corresponding `AnalyticsResponse` event."